SUMO-254673 doc changes for v 1 to v 2 migrations apps (#5019)
SumoLogic/sumologic-documentation: docs/integrations/containers-orchestration/rabbitmq.md

This section provides instructions for configuring log and metric collection for the Sumo Logic App for RabbitMQ.

### Step 1: Fields in Sumo Logic

The following [fields](https://help.sumologic.com/docs/manage/fields/) will always be created automatically as a part of the app installation process:

* `pod_labels_component`
* `pod_labels_environment`
* `pod_labels_messaging_system`
* `pod_labels_messaging_cluster`

> If you're using RabbitMQ in a non-Kubernetes environment, these additional fields will get created automatically as a part of the app installation process:

Reviewer: ```suggestion
If you're using RabbitMQ outside of a Kubernetes environment, the following additional fields will be automatically created during the app installation process:
```

> If you're using RabbitMQ in a non-Kubernetes environment, these additional fields will get created automatically as a part of the app installation process:

Reviewer: ```suggestion
If you're using RabbitMQ outside of a Kubernetes environment, the following additional fields will be automatically created during the app installation process:
```

kubectl describe pod <RabbitMQ_pod_name>
```
   5. Sumo Logic Kubernetes collection will automatically start collecting logs from the pods having the annotations defined above.

> 3. **FER to normalize the fields in Kubernetes environments**. Labels created in Kubernetes environments automatically are prefixed with `pod_labels`. To normalize these for our app to work, a Field Extraction Rule is automatically created named **AppObservabilityMessagingRabbitMQFER**

Reviewer: ```suggestion
3. **FER to normalize the fields in Kubernetes environments**. Labels created in Kubernetes environments automatically are prefixed with `pod_labels`. To normalize these for our app to work, a Field Extraction Rule named **AppObservabilityMessagingRabbitMQFER** is automatically created.
```

> 3. **FER to normalize the fields in Kubernetes environments**. Labels created in Kubernetes environments automatically are prefixed with `pod_labels`. To normalize these for our app to work, a Field Extraction Rule is automatically created named **AppObservabilityMessagingRabbitMQFER**

Reviewer: ```suggestion
3. **FER to normalize the fields in Kubernetes environments**. Labels created in Kubernetes environments automatically are prefixed with `pod_labels`. To normalize these for our app to work, a Field Extraction Rule named **AppObservabilityMessagingRabbitMQFER** is automatically created.
```
